webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -136,6 +136,86 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     await fetchExperiments();
   }, [fetchExperiments]);
 
+  const handleUpdateExperimentList = useCallback(
```

Comment:
nit: Claire showed me [this comparison](https://gist.github.com/ClaireNeveu/d6dd36a126d1ece9d5fcfd769be964ac) after I did the bulk action work and I agree that the second approach there is easier to read, take a look and see what you think

Reply:
Thanks for sharing, I refactored the `handleUpdateExperimentList` function, could you please take another look to verify